# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../apr-util/apu-config.patch.cross
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- apr-util-1.3.2/apu-config.in.vanilla	2009-11-20 18:49:59.000000000 +0100
+++ apr-util-1.3.2/apu-config.in	2009-11-20 18:58:41.000000000 +0100
@@ -81,7 +81,7 @@
 fi
 
 if test "$location" = "installed"; then
-    LA_FILE="$libdir/lib${APRUTIL_LIBNAME}.la"
+    LA_FILE=""
 
     LIBS=`echo "$LIBS" | sed -e "s $APU_BUILD_DIR/xml/expat $prefix g" -e "s $prefix/lib/libexpat.la -lexpat g"`
     LDFLAGS=`echo "$LDFLAGS" | sed -e "s $APU_BUILD_DIR/xml/expat $prefix g"`
@@ -135,7 +135,7 @@
     ;;
     --includes)
     if test "$location" = "installed"; then
-        flags="$flags -I$includedir $INCLUDES"
+        flags="$flags $INCLUDES"
     elif test "$location" = "source"; then
         flags="$flags -I$APU_SOURCE_DIR/include $INCLUDES"
     else
@@ -157,7 +157,7 @@
     --link-ld)
     if test "$location" = "installed"; then
         ### avoid using -L if libdir is a "standard" location like /usr/lib
-        flags="$flags -L$libdir -l$APRUTIL_LIBNAME"
+        flags="$flags -l$APRUTIL_LIBNAME"
     else
         flags="$flags -L$APU_BUILD_DIR -l$APRUTIL_LIBNAME"
     fi
@@ -173,7 +173,7 @@
         ### avoid using -L if libdir is a "standard" location like /usr/lib
         # Since the user is specifying they are linking with libtool, we
         # *know* that -R will be recognized by libtool.
-        flags="$flags -L$libdir -R$libdir -l$APRUTIL_LIBNAME"
+        flags="$flags -l$APRUTIL_LIBNAME"
     else
         flags="$flags $LA_FILE"
     fi
